scripts/release.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+start() {
+    release_number=$1
+    echo -e "Starting release ${release_number}"
+    create_issue
+    create_release_branch
+}
+
+create_issue() {
+    command curl -s \
+        -X POST \
+        -H "Accept: application/json" \
+        -H "Authorization: token ${GITHUB_TOKEN}" \
+        https://api.github.com/repos/airyhq/airy/issues \
+        -d "{\"title\":\"Release ${release_number}\", \"labels\": [\"release\"]}" | jq '.number'
+}
+
+create_release_branch() {
+    command git checkout develop
+    command git pull origin develop
+    command git checkout -b release/${release_number}
+    command git push origin release/${release_number}
+}
+
+finish() {
+    release_number=$1
+    echo -e "Finishing release ${release_number}"
+    increase_version
+    commit_version
+    merge_main
+    merge_develop
```

Comment:
When do we close the issue?

Reply:
it should close automatically based on the commit message?